test/paper-menu.html

```diff
@@ -0,0 +1,69 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+
+    <title>paper-menu tests</title>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../test-fixture/test-fixture-mocha.js"></script>
+
+    <link rel="import" href="../../test-fixture/test-fixture.html">
+    <link rel="import" href="../paper-menu.html">
+
+  </head>
+  <body>
+
+    <test-fixture id="basic">
+      <template>
+        <paper-menu>
+          <div>item 1</div>
+          <div>item 2</div>
+          <div>item 3</div>
+        </paper-menu>
+      </template>
+    </test-fixture>
+
+    <script>
+
+      suite('menu a11y tests', function() {
+        var menu, selectedMenu;
+
+        setup(function() {
+          menu = fixture('basic');
+        });
+
+        test('menu has role="menu"', function() {
+          assert.equal(menu.getAttribute('role'), 'menu', 'has role="menu"');
+        });
+
+        test('first item gets focus when menu is focused', function() {
+          menu.focus();
+          assert.equal(document.activeElement, menu.firstElementChild, 'document.activeElement is first item')
+        });
+
+        test('selected item gets focus when menu is focused', function() {
```

Comment:
:sparkles: This is cool. I think we should find a way to generalize this behavior somehow. No action required, just thinking about how it could be useful elsewhere.